templates/components/pattern-card.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Simple wrapper used when an external Twig template is specified using the 'use:' property.
+ *
+ * @see \Drupal\ui_patterns\Element\Pattern::processUse().
+ */
+#}
+{# until interface options are available, all cards will be icon cards on SAA sites #}
+{% set attributes = attributes.addClass('su-card--icon pt-0 text-left') %}
+{#{% set card_image_class = card_image_class.addClass('saa-icon-card--image') %}#}
+
+{% include use %}
```

Comment:
Oh, this is trixy. 

https://github.com/nuvoleweb/ui_patterns/blob/8.x-1.x/src/Element/Pattern.php#L132

From what I gather by that code it uses the template defined in the pattern yml. Neat.